submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/api/FilesService.kt

```diff
@@ -9,4 +9,10 @@ internal interface FilesService {
     fun postProcessSubmissionFiles(sub: ExtSubmission)
 
     fun cleanSubmissionFiles(previous: ExtSubmission, current: ExtSubmission?)
+
+    fun deleteSubmissionFiles(sub: ExtSubmission)
+
+    fun deleteSubmissionFile(sub: ExtSubmission, file: ExtFile)
```

Comment:
having the same method two times make me think that there are two clases implementing the same logic. as 

 deleteSubmissionFiles(sub: ExtSubmission) => Iterate and call deleteSubmissionFile(sub: ExtSubmission, file: ExtFile)
 
 So if we offer ` fun deleteSubmissionFiles(sub: ExtSubmission)` is because another class is doing exactly the same iteration and deleteing all files.  I hard to see it right now but it looks to me we need a single abtractions and either caller class control flow or this class control it.

Reply:
yes, the other one will disappear in the next PR